lms/templates/preview_menu.html

```diff
@@ -26,10 +26,30 @@
     staff_selected = selected(not masquerade or masquerade.role != "student")
     specific_student_selected = selected(not staff_selected and masquerade.user_name)
     student_selected = selected(not staff_selected and not specific_student_selected and not masquerade_group_id)
+
+    if settings.HTTPS == 'on':
+        protocol = 'https'
+    else:
+        protocol = 'http'
+    insights_base_url = settings.ANALYTICS_DASHBOARD_URL or 'http://localhost:18000/courses'
+    studio_base_url = protocol + '://' + settings.CMS_BASE
+    insights_url = insights_base_url
+    studio_url = studio_base_url
+    if course and course.id:
+        if insights_base_url:
+            insights_url = "{base_url}/{course_id}".format(
+                base_url=insights_base_url,
+                course_id=course.id,
+            )
+        if studio_base_url:
+            studio_url = "{base_url}/course/{course_id}".format(
+                base_url=studio_base_url,
+                course_id=course.id,
+            )
     %>
     <nav class="wrapper-preview-menu" aria-label="${_('Course View')}">
         <div class="preview-menu" style="display: flex; align-items: center;">
```

Comment:
Could I suggest changing the `align-items` here to `start` instead of `center`?  I think having the dropdown aligned to the top when the window width is shrunken down will look a bit cleaner.

Reply:
Otherwise, I screwed around with this for a while just now and came up with much the same layout.

Reply:
Hrmm, it _might_ look a little better when shrunken down, but I think it may look worse when wide, since it's all on a single line and no longer centered vertically on the one side.

## `center`, shrunk

![Screenshot_2020-06-02_08-10-24](https://user-images.githubusercontent.com/7300975/83559968-4a974c80-a4ca-11ea-8591-6f3a835a2254.png)

## `start`, shrunk

![Screenshot_2020-06-02_12-09-20](https://user-images.githubusercontent.com/7300975/83559892-2cc9e780-a4ca-11ea-8aa3-60461862e5b5.png)

## `start`, wide

![Screenshot_2020-06-02_12-07-14](https://user-images.githubusercontent.com/7300975/83559925-394e4000-a4ca-11ea-8828-d4a259a1eceb.png)

## `center`, wide

![Screenshot_2020-06-02_08-09-27](https://user-images.githubusercontent.com/7300975/83560038-68fd4800-a4ca-11ea-83c6-57796c7d511c.png)

Reply:
Oh, darn.  Yup, looks like junk with `start`.  Ah well.